mainwindow.cpp

```diff
@@ -31,6 +31,7 @@ MainWindow::MainWindow(QWidget *parent)
     schedule.groupSchedule = groupSchedule;
     utilityDb -> insertScheduleToTable("KN_31", schedule);
     Schedule schedule1 = utilityDb ->getScheduleByTableName("KN_31");
```

Comment:
Why do use this style "KN_31" but not "KN-31" directly?

Reply:
Seriously?) Name of table can’t consist of “-“ in SQLite

Reply:
Ohh, it's a new discovery for me, really..